packages/components/src/date-time/style.scss

```diff
@@ -1,5 +1,7 @@
 // We can't reference this package with ~ because of how Lerna handles packages. 😩
+/*rtl:begin:ignore*/
```

Comment:
It might be good to have an explanation in a comment here and a link to the issue.